digitalpy/core/persistence/value_change_event.py

```diff
@@ -1,5 +1,5 @@
-from digitalpy.core.event import Event
-from digitalpy.model.persistent_object import PersistentObject
+from digitalpy.core.util.event import Event
+from digitalpy.core.persistence.persistent_object import PersistentObject
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position